packages/twenty-front/src/modules/settings/security/components/SSO/SettingsSSOSAMLForm.tsx

```diff
@@ -136,6 +137,7 @@ export const SettingsSSOSAMLForm = () => {
             Icon={IconUpload}
             onClick={handleUploadFileClick}
             title={t`Upload file`}
+            type="button"
```

Comment:
looks to be the default (type='button') do we need to specify it everytime?

Reply:
The default is `type="submit"` 😅 The change makes sense if the button is used in a `<form>` HTML element and mustn't trigger the form submission. @AMoreaux, do you confirm the use case of these buttons?

Reply:
Yes, in a form, all buttons without a type are submit buttons. I added the type button to avoid triggering the submission of the form.